src/core/mac/link_raw.hpp

```diff
@@ -237,13 +242,31 @@ class LinkRaw
      */
     otError SetExtAddress(const otExtAddress &aExtAddress);
 
+    /**
+     * This method gets the transmit frame.
+     *
+     * @returns A pointer to the transmit frame.
+     *
+     */
+    otRadioFrame *GetTransmitFrame(void) { return mTransmitFrame; }
+
 private:
-    otInstance &mInstance;
+    enum
+    {
+        kOperationTransmitData = 1 << 0,
+    };
+
+    void        TransmitNow(void);
+    void        StartTransmit(void);
+    static void HandleOperationTask(Tasklet &aTasklet);
+    void        HandleOperationTask();
```

Comment:
All void functions or methods shall explicitly declare and specify the void type keyword.